Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -462,8 +462,14 @@ StyleAppearance RenderTheme::autoAppearanceForElement(RenderStyle& style, const
                 return StyleAppearance::SearchFieldResultsButton;
         }
 
-        if (part == UserAgentParts::webkitSliderThumb())
+        if (part == UserAgentParts::webkitSliderThumb()) {
+            if (RefPtr shadowHost = elementPtr->shadowHost()) {
+                if (CheckedPtr shadowStyles = shadowHost->computedStyle(); shadowStyles
```

Comment:
This should probably use `existingComputedStyle()` over `computedStyle()` so that we don't try and compute during style adjustment.

Would also like to hear @anttijk's thoughts on checking `existingComputedStyle()` of a parent node during style adjustment.